Sources/Public/Item Views/DayView.swift

```diff
@@ -302,7 +302,7 @@ extension DayView {
         color = .lightGray
       }
 
-      return DrawingConfig(fillColor: color, borderColor: color)
+      return DrawingConfig(fillColor: color, borderColor: .clear)
```

Comment:
Prevents an issue where a transparent color is used, which would make the border visible since it's drawn over the fill color.